binaries/daemon/src/lib.rs

```diff
@@ -275,6 +278,31 @@ impl Daemon {
                 }
                 (None, RunStatus::Continue)
             }
+            DaemonCoordinatorEvent::Logs {
+                dataflow_id,
+                node_id,
+            } => {
+                // read file
+                let logs = async {
+                    let log_dir = temp_dir();
+
+                    let mut file = File::open(log_dir.join(format!("{dataflow_id}-{node_id}.txt")))
```

Comment:
We currently specify the file name in two places. It would be cleaner to extract this into a separate function. I.e. the function would take the `dataflow_id` and `node_id` as inputs and return `format!("{dataflow_id}-{node_id}.txt"`.

Reply:
Agreed.